en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
 name: Ranking Criteria
 ---
 
-
 # Quaver Ranking Criteria
 
-In order to get a mapset ranked, it must follow the criteria listed below. In the event that it does not, you may be asked to modify your mapset and resubmit it again for rank.
+In order to get a mapset ranked, it must follow the ranking criteria listed below. In the event of it breaking the criteria you may be asked to modify your mapset.
+
+The ranking criteria is divided into rules and guidelines.
+* Rules are, literally, rules that may not be broken under any circumstances.
+* Guidelines try to guide the mapper and may be ignored when deemed reasonable.
+
+If you're looking for the steps of getting a mapset ranked, see: [Quaver Ranking Process](/docs/Ranking/Process).
+
+# Metadata
+
+#### Rules
+
+* **Metadata must be consistent across difficulties.** The only exception being specific map-descriptive tags, such as pattern names, that improve the searchability of singular difficulties.
+
+## Artist
+
+#### Rules
+
+* **The artist must be absolutely correct.**
+* **The artist must be romanized.** Non-romanized artist can be added to tags.
+* **The artist must not repeat in tags.** Only exception being the case where the artist appears in a relevant tag, such as song genre.
+* If there is no existing person to credit as the artist, `Unknown Artist` must be used.
+
+## Title
+
+#### Rules
+
+* **The title must be absolutely correct.**
+* **The title must be romanized.** Non-romanized title can be added to tags. Loan words from other languages should use the original words instead of attempted romanization.
+* **The title must not repeat in tags.** Only exception being the case where the song title appears in a relevant tag, such as song genre.
+
+## Source
+
+#### Rules
+
+* **The source field must be used if the song is directly from a video game, album, series etc.**
+* **Remixes, arrangements, covers etc. that are based on another song should use the original song's source.**
```

Comment:
In addition to this: How would we handle songs that sample other material, but are given proper releases on different albums or other projects entirely unrelated to the original material?

Some examples of what I'm referring to:
https://quavergame.com/mapset/map/62060
https://quavergame.com/mapset/map/70635

You could interpret these songs as being unique songs that simply sample other material rather than being remixes - especially when they are given unique song titles with no indication of which artists are being sampled.

Reply:
Personally I'd just have stuff about the sampled material in tags, which I think is what's been happening already with DJ Witch Girl Miracle's stuff.

I'd also have anything regarding this as a guideline, as it seems it would be pretty hard to actually enforce in the cases of the sampling not being indicated.

Reply:
Agree on putting sampled material in tags.

DJ Witch Girl Miracle is a bit of a different case as they don't make their music themselves (nor do they claim to as far as I can tell) - they just DJ and upload nightcore edits (some also have cut intros/outros or similar). If you download the original songs that are linked in their uploads and speed them up accordingly, you'll be left with the exact same product.

But that's also a perfect example of how hard it would be to enforce, when it seems to sometimes be difficult even when having the correct metadata right infront of us.